From fa35e3823231d363f1fe35b936614a2ff32ec13e Mon Sep 17 00:00:00 2001
From: Vicentiu Galanopulo <vicentiu@balena.io>
Date: Thu, 5 Mar 2020 09:30:20 +0100
Subject: [PATCH] Disable HDMI EDID from the dts node

In order to use one of the defined video modes of the IMX8
edid must be disabled:

https://github.com/compulab-yokneam/Documentation/wiki/iMX8M-HDMI

Upstream-Status: Inappropriate [configuration]
Signed-off-by: Vicentiu Galanopulo <vicentiu@balena.io> 
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 669f71201532..8e8c1e562f0e 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -589,7 +589,7 @@
 		interrupts = <GIC_SPI 16 IRQ_TYPE_LEVEL_HIGH>,
 					<GIC_SPI 25 IRQ_TYPE_LEVEL_HIGH>;
 		interrupt-names = "plug_in", "plug_out";
-		fsl,cec;
+		fsl,no_edid;
 		status = "disabled";
 
 		port@0 {
-- 
2.17.1

